functions/Restore-DbaDatabase.ps1

```diff
@@ -178,8 +178,31 @@ folder for those file types as defined on the target instance.
         [string]$DestinationFilePrefix=''					
 	)
     BEGIN
-    {
-        $FunctionName = $FunctionName =(Get-PSCallstack)[0].Command
+	{
+		$base = $SqlServer.Split("\")[0]
+		
+		if ($base -eq "." -or $base -eq "localhost" -or $base -eq $env:computername -or $base -eq "127.0.0.1")
+		{
+			# they can use whatever path they like
+		}
+		else
+		{
+			if ($Path.StartsWith("\\") -eq $false)
+			{
+				# Many internal functions parse using Get-ChildItem. 
+				# We need to use Test-SqlPath and other commands instead
+				# Prevent people from trying 
+				
+				Write-Warning "Currently, you can only use UNC paths when running this command remotely. We expect to support local paths shortly."
+				continue
```

Comment:
Not in BEGIN. Try it. It stops everything. If you use return in BEGIN, it moves on to PROCESS